### PR TITLE
DOCS: Adding Abbrivation

### PR DESCRIPTION
**Pull request by bobbyy16**
Closes #

### What new changes did you make? Tick all applicable boxes
- [x] Added new abbreviation
- [ ] Fixed something in the source code
- [ ] Added a new feature
- [ ] Fixed the docs (README.md, CONTRIBTUING.md, etc)

### CODE REVIEW

File: public/server/db.json [hunk 1]
```diff
@@ -72,6 +72,10 @@
   "bff": {
     "definition": "Best Friend Forever"
   },
+  "bf": {
```
Comment (Njong392) [inline public/server/db.json:75]: You didn't need to additionally add this to `db.json`. We no longer use this file to store abbreviations.

File: public/server/db/bf.json [hunk 2]
```diff
@@ -0,0 +1,4 @@
+
+{
+  "definition": "Boy Friend"
```
Comment (Njong392) [inline public/server/db/bf.json:3]: In the other file, you added an alternatives field. But you left that out here. Would you like to add that before I merge this in?
```suggestion
  "definition": "Boy Friend",
  "alternatives": "Breakfast"
```

**Review by Njong392 — CHANGES_REQUESTED** [review]
Thank you so much for the stellar addition! Please check out some of the changes I have proposed.

**Comment by bobbyy16** [comment]
hey @Njong392 didn't see that one, commited alternative now, thanks for the help

**Review by Njong392 — APPROVED** [review]
Stellar addition! Thank you so much for the new addition. 🎉 

The word boyfriend, however, is one word. Sorry for not picking that tiny detail up.

**Pull request merged**